.github/workflows/build_and_test.yml

```diff
@@ -668,7 +668,15 @@ jobs:
       - name: Build ToyVpn-PcapPlusPlus
         working-directory: ./ToyVpn-PcapPlusPlus
         run: |
+          NDK_VERSION=$(echo "$ANDROID_NDK" | awk -F'/' '{print $NF}')
```

Comment:
This line takes `$ANDROID_NDK` which is [the path of Android NDK](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md#environment-variables-2) and extracts the NDK version, i.e: `/usr/local/lib/android/sdk/ndk/27.0.12077973` ==> `27.0.12077973`